mixins/interactive-mixin.js

```diff
@@ -70,17 +77,19 @@ export const InteractiveMixin = superclass => class extends LocalizeCoreElement(
 
 		return html`
 			<div class="${classMap(classes)}" @keydown="${this._handleInteractiveKeyDown}">
-					<button
-						class="interactive-toggle d2l-offscreen"
-						@blur="${this._handleInteractiveToggleBlur}"
-						@click="${this._handleInteractiveToggleClick}"
-						@focus="${this._handleInteractiveToggleFocus}"
-						tabindex="${ifDefined(this._hasInteractiveAncestor && !this._interactive ? '0' : '-1')}">
-						${`${label}, ${this.localize('components.interactive.instructions')}`}
-					</button>
+				<button
+					class="interactive-toggle d2l-offscreen"
+					@blur="${this._handleInteractiveToggleBlur}"
+					@click="${this._handleInteractiveToggleClick}"
+					@focus="${this._handleInteractiveToggleFocus}"
+					tabindex="${ifDefined(this._hasInteractiveAncestor && !this._interactive ? '0' : '-1')}">
+					${`${label}, ${this.localize('components.interactive.instructions')}`}
+				</button>
+				<div class="interactive-trap">
```

Comment:
This extra `div` just makes it easier for the above helper method to detect when focus has moved into the content (excluding the `button`).